docs/src/main/asciidoc/security-oidc-bearer-token-authentication.adoc

```diff
@@ -99,19 +103,20 @@ The default tenant's `OidcConfigurationMetadata` is injected if the endpoint is
 [[token-claims-and-security-identity-roles]]
 === Token Claims And SecurityIdentity Roles
 
-SecurityIdentity roles can be mapped from the verified JWT access tokens as follows:
+You can map `SecurityIdentity` roles from the verified JWT access tokens as follows:
 
-* If `quarkus.oidc.roles.role-claim-path` property is set and matching array or string claims are found then the roles are extracted from these claims.
-  For example, `customroles`, `customroles/array`, `scope`, `"http://namespace-qualified-custom-claim"/roles`, `"http://namespace-qualified-roles"`, etc.
-* If `groups` claim is available then its value is used
-* If `realm_access/roles` or `resource_access/client_id/roles` (where `client_id` is the value of the `quarkus.oidc.client-id` property) claim is available then its value is used.
-  This check supports the tokens issued by Keycloak
+* If the `quarkus.oidc.roles.role-claim-path` property is set, and matching array or string claims are found, then the roles are extracted from these claims.
+For example, `customroles`, `customroles/array`, `scope`, `"http://namespace-qualified-custom-claim"/roles`, `"http://namespace-qualified-roles"`.
+* If a group's claim is available, then its value is used.
+* If a `realm_access/roles` or `resource_access/client_id/roles` (where `client_id` is the value of the `quarkus.oidc.client-id` property) claim is available, then its value is used.
+This check supports the tokens issued by Keycloak.
 
-If the token is opaque (binary) then a `scope` property from the remote token introspection response will be used.
+If the token is opaque (binary), then a `scope` property from the remote token introspection response is used.
 
-If UserInfo is the source of the roles then set `quarkus.oidc.authentication.user-info-required=true` and `quarkus.oidc.roles.source=userinfo`, and if needed, `quarkus.oidc.roles.role-claim-path`.
+If `UserInfo` is the source of the roles, then set `quarkus.oidc.authentication.user-info-required=true` and `quarkus.oidc.roles.source=userinfo`, and if needed, set `quarkus.oidc.roles.role-claim-path`.
 
-Additionally, a custom `SecurityIdentityAugmentor` can also be used to add the roles as documented in xref:security-customization.adoc#security-identity-customization[Security Identity Customization].
+A custom `SecurityIdentityAugmentor` can also be used to add the roles.
+For more information, see the xref:security-customization.adoc#security-identity-customization[Security Identity Customization] section of the Quarkus "Security tips and tricks" guide.
```

Comment:
Minor/non-urgent: Capitalization of the URL text. Should this be "Security identity customization"